### PR TITLE
feat-4868: Delete the configurations of the master and slave nodes.

### PR DESCRIPTION
**Pull request by casionone**
### What is the purpose of the change
Remove the configuration of the master and slave nodes.

1. LinkisManager refreshes all engine configurations wds.linkis.engineconn.dist.load.enable=true
2. AM service matrix monitor configuration wds.linkis.manager.am.monitor.switch.on=false
3. ps-cs registry label configuration spring.eureka.instance.metadata-map.route=cs_1_dev

Remove the above configuration and code control master or slave nodes.

### Related issues/PRs

Related issues: #254
Related pr: #591

### CODE REVIEW

File: linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/engineplugin/server/service/DefaultEngineConnResourceService.scala [hunk 1]
```diff
@@ -58,15 +59,36 @@ class DefaultEngineConnResourceService extends EngineConnResourceService with Lo
...
-    if (EngineConnPluginConfiguration.ENGINE_CONN_DIST_LOAD_ENABLE.getValue) {
-      logger.info("Start to refresh all engineconn plugins when inited.")
+  override def init(): Unit = {
+    commonLock.setLockObject(_LOCK)
+    commonLock.setCreateTime(new Date)
+    commonLock.setUpdateTime(new Date)
+    commonLock.setCreator(Utils.getJvmUser)
+    commonLock.setUpdator(Utils.getJvmUser)
+    lock = commonLockService.lock(commonLock, -1)
+    if (lock) {
```
Comment (peacewong) [inline linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/engineplugin/server/service/DefaultEngineConnResourceService.scala:79]: 需要确保可以重入该锁，如果服务是OOM，destory是不会执行的

File: linkis-computation-governance/linkis-manager/linkis-application-manager/pom.xml [hunk 2]
```diff
@@ -95,6 +95,18 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.linkis</groupId>
+      <artifactId>linkis-instance-label-client</artifactId>
+      <version>${project.version}</version>
```
[the remark this answers is not in the document]
Comment (casionone) [reply linkis-computation-governance/linkis-manager/linkis-application-manager/pom.xml:101]: refresh和remove label的时候会用到这里面的代码
InstanceLabelClient.getInstance.refreshLabelsToInstance(insLabelRefreshRequest)
InstanceLabelClient.getInstance.removeLabelsFromInstance(insLabelRemoveRequest)

File: linkis-public-enhancements/linkis-ps-common-lock/src/main/resources/mapper/common/CommonLockMapper.xml [hunk 3]
```diff
@@ -21,15 +21,18 @@
...
     <delete id="unlock">
-        delete  from linkis_ps_common_lock where lock_object = #{jsonObject}
+        delete  from linkis_ps_common_lock where lock_object = #{commonLock.lockObject} and creator = #{commonLock.creator}
     </delete>
 
     <select id="getAll" resultType="org.apache.linkis.publicservice.common.lock.entity.CommonLock">
         select * from linkis_ps_common_lock
     </select>
+    <select id="getLock" resultType="org.apache.linkis.publicservice.common.lock.entity.CommonLock">
+        select * from linkis_ps_common_lock where lock_object = #{commonLock.lockObject} and creator = #{commonLock.creator}
```
Comment (casionone) [inline linkis-public-enhancements/linkis-ps-common-lock/src/main/resources/mapper/common/CommonLockMapper.xml:36]: 函数命名太通用 但是sql并不是一个通用的场景 考虑修改下名
加host 字段方式更好点？

File: linkis-dist/package/db/linkis_ddl.sql [hunk 4]
```diff
@@ -136,6 +136,7 @@ DROP  TABLE IF EXISTS `linkis_ps_common_lock`;
 CREATE TABLE `linkis_ps_common_lock` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `lock_object` varchar(255) COLLATE utf8_bin DEFAULT NULL,
+  `host` varchar(255) COLLATE utf8_bin NOT NULL,
```
Comment (casionone) [inline linkis-dist/package/db/linkis_ddl.sql:139]: 字符集需要调整

**Review by peacewong — APPROVED** [review]

**Review by aiceflower — APPROVED** [review]

**Pull request merged**